src/scenarios/blazor/test.py

```diff
@@ -0,0 +1,13 @@
+'''
+Empty Blazor Wasm Template
+'''
+from shared.runner import TestTraits, Runner
+
+SCENARIONAME = 'Empty Blazor Wasm Template'
```

Comment:
this isn't actually used right? I know it's still hanging out in some other files but we shouldn't keep including it if it isn't doing anything.

Reply:
Agree. I have lost track of why it's there ( maybe originally put there as a metadata ?) but it's not used and duplicate info from the comment on the top.